syntax = "proto3";


import "services/commands/setStateOutput.proto";
import "services/commands/linkInputToOutput.proto";



service DeviceService {
  rpc SetStateOutput (SetStateOutputRequest) returns (SetStateOutputResponse);
  rpc LinkInputToOutput (LinkInputToOutputRequest) returns (LinkInputToOutputResponse);
}

